internal/gps/result.go

```diff
@@ -51,11 +52,15 @@ type solution struct {
 // It requires a SourceManager to do the work, and takes a flag indicating
 // whether or not to strip vendor directories contained in the exported
 // dependencies.
-func WriteDepTree(basedir string, l Lock, sm SourceManager, sv bool) error {
+func WriteDepTree(basedir string, l Lock, sm SourceManager, sv bool, logger *log.Logger) error {
 	if l == nil {
 		return fmt.Errorf("must provide non-nil Lock to WriteDepTree")
 	}
 
+	if logger != nil {
```

Comment:
🤔 I am not really sure. We haven't settled on a standard yet, but what you see on ctx is what we have been doing so far. 

I suggest that for now, follow the other examples in dep for verbose logging. If after you've done that, you see a way to simplify verbose logging, let's look at that in a separate issue or PR.